components/inputs/input-number.js

```diff
@@ -63,14 +95,18 @@ class InputNumber extends LitElement {
 				class="d2l-input"
 				?disabled="${this.disabled}"
 				id="${this._inputId}"
-				max="${ifDefined(this.max)}"
-				min="${ifDefined(this.min)}"
 				name="${ifDefined(this.name)}"
 				placeholder="${ifDefined(this.placeholder)}"
-				step="${ifDefined(this.step)}"
```

Comment:
I'm assuming you removed `max`, `min`, and `step` because they don't work well with the formatted value?

Reply:
So I removed `max`, `min`, and `step` values because they have no function with a 'text input' element. However, I still handle `min` and `max` values with the `checkMinMaxValue` method.